optuna/storages/rdb/storage.py

```diff
@@ -1009,6 +1009,12 @@ def get_n_trials(self, study_id, state=None):
         self._commit(session)
         return n_trials
 
+    def read_from_remote_storage(self, study_id: int) -> None:
+        # Check that that study exists.
+        session = self.scoped_session()
+        models.StudyModel.find_or_raise_by_id(study_id, session)
```

Comment:
It might be helpful to note that `KeyError` is raised if the study for the given `study_id` does not exist as follows.
```suggestion
        # If the study for the ``study_id`` does not exist, `KeyError` is raised.
        models.StudyModel.find_or_raise_by_id(study_id, session)
```

Reply:
It's documented in the `BaseStorage` doc and I think we don't need the verbose comment here.

Reply:
I see. Thanks for your correction.